src/components/App.js

```diff
@@ -45,6 +46,7 @@ componentDidMount = () => {
           <h1 className='siteTitle'> 🍿 Rancid Tomatillos 🍿</h1>
         </header>
         {/* <SearchBar movies={this.state.movies} /> */}
+        { this.state.isLoading && <p>⏳ Loading...</p> }
```

Comment:
Nice job using conditional rendering to let the user know what's going on if all the movies images take a bit longer to load! ⏳